openapi.yml

```
@@ -1628,7 +1677,16 @@ components:
         id: 8oa3c4w7ml7tx
       type: object
       additionalProperties:
-        type: string
+        type: array
+        items:
+          type: string
+    CreateRequestWithSubRequest:
```

nitpick: CreateRequestGroup might be more concise here

```suggestion
    CreateRequestGroup:
```